package.json

```
@@ -157,6 +157,7 @@
 		"@woocommerce/customer-effort-score": "file:packages/customer-effort-score",
 		"@woocommerce/data": "file:packages/data",
 		"@woocommerce/date": "file:packages/date",
+		"@woocommerce/dependency-extraction-webpack-plugin": "^1.4.0",
```

🤔 this package is one of ours, thats in `packages`. I'm not sure we need to list this dependency, but if we did we should perhaps reference the in-repo package like we have with others.

```suggestion
		"@woocommerce/dependency-extraction-webpack-plugin": "file:packages/dependency-extraction-webpack-plugin",
```